public_html/wp-content/mu-plugins/blocks/source/blocks/speakers/index.js

```diff
@@ -6,6 +6,7 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import './style.scss';
```

Comment:
`SHOULD` - My first thought would be to move in the other direction, so `sessions/style.scss -> sessions/block-content.scss`. The Schedule block will have both `block-content.scss` and `inspector-controls.scss`, and I think it'll be harder to  work with if they're combined into a single file.

Reply:
I haven't officially proposed standardizing this, but if we want to follow gutenberg (and eventually get the benefit of having separate editor & general styles), [we should have max 2 .scss files per folder,](https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#scss-file-naming-conventions-for-blocks) one `editor.scss` and one `style.scss`. This is enforced by GB's build process, we could look into doing the same, or opting for the [slightly more complicated version](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/0ac1ba488815acc0abfd25451c91652f9c289fa0/webpack.config.js#L78-L95) I used in the Woo blocks.

Reply:
+1 for sticking with `style.scss` for now, and eventually moving to a style/editor split like in Gutenberg. I tried to implement that in our original custom webpack config, but I couldn't figure out how to do it efficiently.

Reply:
Ah, splitting between editor and front end sounds like a compelling reason.

Is there a way to prevent duplicating styles, though? If we'd have to duplicate them then I'd be more hesitant.

Reply:
It's not split between editor & front end, it's shared styles (style.scss) and editor-specific styles (editor.scss). If we need to introduce a third, front-end only file I think there's precedence for calling it `theme.css`, but I'm not sure if that's meant to be theme-specific (can look into it more if we need to).

Reply:
Doh, thanks for pointing that out. This sounds good to me then.